src/containers/tutor-home-page/language-step/LanguageStep.jsx

```diff
@@ -1,16 +1,62 @@
+import React, { useState } from 'react'
 import Box from '@mui/material/Box'
+import { Typography, Autocomplete, TextField } from '@mui/material'
 
 import { styles } from '~/containers/tutor-home-page/language-step/LanguageStep.styles'
 import img from '~/assets/img/tutor-home-page/become-tutor/languages.svg'
 
+import { languagesMock } from '../subjects-step/constants'
+
+const languages = languagesMock.map((lang) => lang.name)
+
 const LanguageStep = ({ btnsBox }) => {
+  const [selectedLanguage, setSelectedLanguage] = useState(null)
+  const [displayedLanguages, setDisplayedLanguages] = useState(
+    languages.slice(0, 6)
+  )
+
+  const handleScroll = (event) => {
+    const listboxNode = event.currentTarget
+    if (
+      listboxNode.scrollTop + listboxNode.clientHeight >=
+      listboxNode.scrollHeight
+    ) {
+      setDisplayedLanguages((prev) => [
+        ...prev,
+        ...languages.slice(prev.length, prev.length + 6)
+      ])
+    }
+  }
+
   return (
     <Box sx={styles.container}>
       <Box sx={styles.imgContainer}>
         <Box component='img' src={img} sx={styles.img} />
       </Box>
       <Box sx={styles.rigthBox}>
-        <Box>Language here</Box>
+        <Typography variant='h6'>
+          Please select the language in which you would like to study and
+          cooperate.
```

Comment:
Add this text to the constants `src/constants/translations/en/become-tutor.json` and use it `{t('becomeTutor.languages.title')}` for title and `{t('becomeTutor.languages.autocompleteLabel')}` for form label